crypto/mem.c

```diff
@@ -122,6 +122,16 @@ WEAK_SYMBOL_FUNC(void, OPENSSL_memory_free, (void *ptr))
 WEAK_SYMBOL_FUNC(size_t, OPENSSL_memory_get_size, (void *ptr))
 WEAK_SYMBOL_FUNC(void *, OPENSSL_memory_realloc, (void *ptr, size_t size))
 
+int CRYPTO_mem_ctrl(int mode) {
+  #ifdef OPENSSL_NO_CRYPTO_MDEBUG
```

Comment:
Nit:
```suggestion
  #if defined(OPENSSL_NO_CRYPTO_MDEBUG)
```

Spacing's also a bit off below.